app/lizard-nxt.js

```diff
@@ -146,6 +152,26 @@ angular.module('lizard-boostrap', ['favourites'])
       );
     };
 
```

Comment:
This commented code, i.e. the old version of getBootstrap() needs to be removed in final version, is only still here for comparison/trouble-shooting